src/core/tests/node_util_tests.cpp

```diff
@@ -0,0 +1,38 @@
+// Copyright (C) 2018-2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+
+#include "openvino/op/parameter.hpp"
+#include "openvino/op/result.hpp"
+#include "openvino/op/split.hpp"
+#include "openvino/op/util/node_util.hpp"
+
+TEST(NodeUtilTests, set_name_parameter) {
+    std::string name = "test_name";
+    auto param = std::make_shared<ov::op::v0::Parameter>(ov::element::i8, ov::Shape{1});
+    ov::op::util::set_name(*param, name);
+    EXPECT_TRUE(0 == name.compare(param->get_friendly_name()));
```

Comment:
Detail
```suggestion
    EXPECT_EQ(name, param->get_friendly_name());
```
In case of error will print not matched names instead the result false instead true.